R/SurveyFit.R

```diff
@@ -180,7 +180,9 @@ SurveyFit <- R6::R6Class(
           dplyr::relocate(by_var, .after="sd")%>%
           dplyr::mutate(method = "mrp")
 
-        lhs_binary <- self$map()$sample()$survey_data() %>%
+          levels(wtd_ests[[by_var]]) <- levels(mrp_ests[[by_var]])
```

Comment:
Can you check if this is needed? (It might be, just hard to tell from glancing at it)

Reply:
Hmm you're right, I think this can go.